products/nginx.md

```diff
@@ -9,6 +9,13 @@ versionCommand: nginx -v
 releaseColumn: true
 releaseDateColumn: true
 changelogTemplate: https://nginx.org/en/CHANGES-__RELEASE_CYCLE__
+purls:
+-   repology: nginx
+-   purl: pkg:deb/debian/nginx
```

Comment:
@hebbet Note for future PRs: The intent of `repology` key is that our code should be able to expand the first (`repology: nginx`) to all of the below automatically.